job/src/main/resources/index.jelly

```diff
@@ -25,5 +25,5 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <div>
-    Defines a new job type for workflows and provides their generic user interface.
+    Defines a new job type for pipelines and provides their generic user interface.
```

Comment:
I think Manuel is right here: Pipeline creates pipelines.  At least it's consistent with the naming scheme from Workflow.  Maybe everywhere we talk about pipeline-as-created-by-Pipeline we call it a "Pipeline job" or slightly more clunky "pipeline-job".